go/exorelay/exorelay_test.go

```diff
@@ -27,16 +29,51 @@ func newExocom() *exocomMock.ExoComMock {
 	return exocom
 }
 
+func WaitFor(condition func() bool, errorMessage string) error {
```

Comment:
This is a very cool method! I am surprised it does not exist as an open source tool or framework method already.